doctor/views.py

```diff
@@ -169,6 +171,28 @@ def make_png_thumbnails_from_range(request) -> HttpResponse:
         return FileResponse(open(filename, "rb"))
 
 
+def xray(request) -> JsonResponse:
+    """Check PDF for bad redactions
+
+    :return: json with bounding boxes and text
+    """
+    form = DocumentForm(request.POST, request.FILES)
+    if not form.is_valid():
+        return JsonResponse(
+            {"error": True, "msg": "Failed validation"}, status=BAD_REQUEST
+        )
+    extension = form.cleaned_data["extension"]
+    if "pdf".casefold() != extension.casefold():
+        return JsonResponse(
+            {"error": True, "msg": "Failed file type"}, status=BAD_REQUEST
+        )
+    results = get_xray(form.cleaned_data["fp"])
+    if results.get("error", False):
+        return JsonResponse(results, status=BAD_REQUEST)
+    cleanup_form(form)
```

Comment:
I don't love how this bit of code can fail to run if we have an unexpected exception, but I recognize that you're following the samples that came before you. It's not the end of the world if it doesn't run, since we'll throw away this docker pod before it becomes full, but, eh, it's messy. Maybe a try/except/finally would be useful here, but if you don't want to bother, I get that too.

Reply:
(sorry, it looks like my comment got attached to four lines instead of one. I am referring to the cleanup_form function, specifically.)